app/knight/knight.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(
+            self,
+            knight_dict: dict
+    ) -> None:
+        self.name = knight_dict.get("name")
+        self.power = knight_dict.get("power")
+        self.hp = knight_dict.get("hp")

```

Comment:
Initialize these attributes with the use of arguments, do not retrieve them from `knight_dict`